server/Database.js

```diff
@@ -694,6 +700,45 @@ class Database {
     })
   }
 
+  /**
+   * Returns true if a custom provider with the given slug exists
+   * @param {string} providerSlug
+   * @return {boolean}
+   */
+  async doesCustomProviderExistWithSlug(providerSlug) {
+    const id = providerSlug.split("custom-")[1]
+
+    if (!id) {
+      return false
+    }
+
+    return !!await this.customMetadataProviderModel.findByPk(id)
+  }
+
+  /**
+   * Removes a custom metadata provider
+   * @param {string} id
+   */
+  async removeCustomMetadataProviderById(id) {
+    // destroy metadta provider
+    await this.customMetadataProviderModel.destroy({
+      where: {
+        id,
+      }
+    })
+
+    const slug = `custom-${id}`;
```

Comment:
Is there a reason to use a slug here instead of just the UUID?

Reply:
it makes it easier to know a provider is a custom one, as they all have this prefix, if it was just an id you could technically have some future provider be prefixed with valid hex characters or something